config/locals_test.go

```diff
@@ -111,6 +111,11 @@ func TestEvaluateLocalsBlockImpossibleWillFail(t *testing.T) {
 	_, err = config.EvaluateLocalsBlock(ctx, file)
 	require.Error(t, err)
 
+	var multiError *errors.MultiError
+	require.True(t, errors.As(err, &multiError))
+
+	err = multiError.WrappedErrors()[0]
+
```

Comment:
```suggestion
        assert.NotErrorAs(t, err, new(config.CouldNotEvaluateAllLocalsError), "Did not get expected error: %s", err)
```

Reply:
Sorry, I missed your comment. The following code (below your changes)

```go
	switch errors.Unwrap(err).(type) { //nolint:errorlint
	case config.CouldNotEvaluateAllLocalsError:
	default:
		t.Fatalf("Did not get expected error: %s", err)
	}
```

Can be replaced with this one line code.

```go
assert.NotErrorAs(t, err, new(config.CouldNotEvaluateAllLocalsError), "Did not get expected error: %s", err)
```

Reply:
What I'm not understanding is that `NotErrorAs` will assert that non of the errors in the error chain match the target, so why use it?

Reply:
Ah, you're right, it should be 

```
assert.ErrorAs(t, err, new(config.CouldNotEvaluateAllLocalsError), "Did not get expected error: %s", err)
```